api/v1/lib/mesos.proto

```diff
@@ -2058,10 +2051,8 @@ message Offer {
     optional Destroy destroy = 6;
     optional GrowVolume grow_volume = 13; // EXPERIMENTAL.
     optional ShrinkVolume shrink_volume = 14; // EXPERIMENTAL.
-    optional CreateVolume create_volume = 8;
```

Comment:
why weren't these removed tag numbers `reserved` upstream to avoid possible future conflicts?

Reply:
https://developers.google.com/protocol-buffers/docs/proto#reserved